git-interactive-rebase-tool.1

```diff
@@ -0,0 +1,26 @@
+.\" Manpage for interactive-rebase-tool.
+.\" Contact dev@mitmaro.ca to correct errors or typos.
+.TH INTERACTIVE\-REBAS 1 "02/02/2018" "v0.5.0" "Git Manual"
+.SH NAME
+git-interactive-rebase-tool \- an improved sequence editor for git
```

Comment:
Remove `git-` from here. Otherwise the command to view the man page would be `man git-interactive-rebase-tool` while the command is just `interactive-rebase-tool`.